### DISCUSSION TITLE
How to use APO?

### DISCUSSION THREAD

**Discussion by dxandtom** [microsoft/sammo#43] [Help]
when i use APO,it will happen:AttributeError: 'Output' object has no attribute 'with_parent'. Did you mean: '_sym_parent'?
And i found in search.py   the "result" in "task" is output ,which is different from using "Paraphrase"

**Comment by t-schn** [comment]
Hi there,

would you mind sharing a minimal reproducible example?

**Reply by dxandtom** [reply]
I have solved this problem. I have not added the implementation of mistakes to the accuracy part. After the implementation, the accuracy part can be used. In addition, I would like to ask that when I want to use APO in BagOfmutators, when there is no error (that is, when the score is full), the entire candidate set is returned directly. Thank you for your reply.

**Reply by t-schn** [reply]
Can you clarify the last part? Also, it might be helpful to look at this example: 
https://github.com/microsoft/sammo/blob/e7a57b5669474bf4c96eb1efacfdf0dc6319abc7/examples/paper_instruction_tuning/instruction_tuning_sammo.py#L187-L203
